clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1299,6 +1377,178 @@ public void testForceMetadataDeleteForPatternSubscriptionDuringRebalance() {
         }
     }
 
+    @Test
+    public void testOnJoinPrepareWithOffsetCommitShouldSuccessAfterRetry() {
+        rebalanceConfig = buildRebalanceConfig(Optional.of("group-id"));
```

Comment:
Why do we need `groupInstanceId`? Could we use class variable: `rebalanceConfig` directly here?

Reply:
fixed, thanks